website/source/docs/templates/user-variables.html.md

```diff
@@ -166,6 +166,9 @@ Results in the following variables:
 | aws_access_key | foo |
 | aws_secret_key | baz |
 
+The `-private` flag will mark as sensitive any variable, regardless of whether
```

Comment:
I don't think private vars are relevant here. Only in the context of packer push.